core/src/test/resources/server.conf

```diff
@@ -141,7 +141,7 @@ cairo.sql.parallel.groupby.enabled=false
 cairo.sql.parallel.groupby.merge.shard.queue.capacity=2048
 cairo.sql.parallel.groupby.sharding.threshold=100
 cairo.sql.parallel.groupby.presize.enabled=false
-cairo.sql.parallel.groupby.presize.max.size=100000
```

Comment:
This looks like an unrelated and hidden change, but it changes the name of a configuration property. Isn't that a potentially breaking change?

Reply:
This property is unlikely to be changed by a user, so while it's a breaking change, it's a low risk one.

Reply:
@mtopolnik my thought process was the same as Andrei's